zstacklib/zstacklib/utils/iscsi_nl.py

```
@@ -0,0 +1,484 @@
+    try:
+        sock.bind((pid, groups))
+    except socket.error as e:
+        logger.error("Failed to bind netlink socket: %s", e)
+        sock.close()
+        return None
+
+    logger.debug("Successfully connected to Netlink with pid=%d, groups=%d", pid, groups)
+    return sock
+
+
+def send_netlink_message(sock, event_type, payload):
```

Pylint (3.3.7)

[refactor] Too many local variables (20/15)

(R0914)